pandas/tests/frame/test_query_eval.py

```diff
@@ -165,7 +164,9 @@ def test_eval_resolvers_as_list(self):
 
 class TestDataFrameQueryWithMultiIndex(tm.TestCase):
 
-    def check_query_with_named_multiindex(self, parser, engine):
+    @pytest.mark.parametrize('parser', ['pandas'])
```

Comment:
Yeah I thought that too. I went with parametrizing simply because it's symmetric with the other ones where we parametrize over all the `PARSERS`. I don't feel strongly about that, if you prefer the other way.